### PR TITLE
Ability to Edit Time Stamp & Implemented Character Limit with Text Adjusting

### PR DESCRIPTION
**Pull request by ChrisCruze**
This pull requests addresses two issues brought up by users. 
1. If Goal TItle is too long, it overlaps with numbers
2. If user forgets to push day before, they want to update time stamp. 

I implemented a character limit on create goal and also added ability to edit a time stamp.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,50 +11,51 @@
...
+    "react-native-web": "^0.11.7",
     "react-navigation": "^4.2.2",
     "react-navigation-stack": "^2.2.3",
     "react-navigation-tabs": "^2.8.2",
     "victory-native": "^34.1.0"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
-    "babel-preset-expo": "~8.0.0",
+    "babel-preset-expo": "^8.1.0",
```
Comment (anandangalig) [inline package.json:58]: @ChrisCruze this change might have tangible effects down the road. Was this required to be done by a new library we installed?

**Comment by anandangalig** [comment]
@ChrisCruze Love the timestamp edit idea and the date-picker!!

I have some feedback:
- After editing and saving, the update is not rendering sometimes. If you want, we can exchange ideas on the solution. It seems we had similar challenges on goals list push process.
- The timestamp two icons (delete and edit) - can we space them out a bit?
- I think the delete timestamp action needs a feedback. Maybe a vibration? Also, do we want **Are your sure?** confirmation before deleting?
- The date-picker popup - I can't click outside of it to cancel. Or perhaps add a Cancel button? It just seems un-intuitive to click on Submit if I want to cancel out. Also, would it look cleaner if the popup was all the way up flush against the header?

These are mere suggestions and my thoughts after clicking through. Let's chat about them if you wanna sync up!

File: Components/Pages/createGoal.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ const createGoal = ({ navigation, goToPassword, createNew }) => {
         placeholder="Title"
         keyboardType="default"
         autoCapitalize="words"
+        maxLength={12}
```
Comment (anandangalig) [inline Components/Pages/createGoal.js:58]: @ChrisCruze I kinda felt like 12 was too short when I creating test goals. Not sure, lol
Also, should we autoCapitalize each word? I liked it when it was autoCapitalize first word. 
This is how they would look: 
- **Call My Mom** 
- **Call my mom**

**Comment by ChrisCruze** [comment]
Great suggestions man. Because of the merge conflicts as well as other reasons, I just re-pulled off master by the way and made the updates in another branch: https://github.com/ChrisCruze/Push-IOS/tree/metrics-edit-timestamp-text-adjust

**Pull request closed without merging**